R/clean_curated_data.R

```diff
@@ -5,63 +5,178 @@ fetch_syn_table <- function(syn_id, cols = "*") {
   if (is.vector(cols)) {
     cols <- paste(cols, collapse = ",")
   }
-  table <- synapser::synTableQuery(paste("select", cols, "from", syn_id))
-  table_df <- table$asDataFrame()
+  table <- synTableQuery(paste("select", cols, "from", syn_id))
+  table_df <- table$asDataFrame() %>%
+    as_tibble()
+  if (has_name(table_df, "ObjectTrackID")) {
```

Comment:
This is duplicated in two places (all 'if's). Can it be abstracted to a function that mutates the data frame?

Reply:
Yes. This is just sloppy coding on my part.